### ISSUE TITLE
[sub-issue] CRw: Alternate texts of Messages & Icons are missing

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#274]
![image](https://cloud.githubusercontent.com/assets/7510842/5203696/6850225e-7589-11e4-8dd2-753326330b4b.png)

The icons are not read by screenreaders.

==> Use ARIA "labelledby" or a hidden text.
If you use ARIA check it out with the screen reader NVDA http://www.nvaccess.org/download/

The Symbols are:
- Success
- Information
- Warning
- Error

**Comment by tonifisler** [comment]
Main issue #340

**Comment by tonifisler** [comment]
As for #275, this depends on the case and should be left for the developer to decide. I've added info about this on the page.
